docs/Mock-Core-Configs.md

```diff
@@ -50,11 +50,13 @@ distribution.
 | [Circle Linux](https://cclinux.org/)                                           | `circlelinux-*`   | [@bella485](https://github.com/bella485)                              | [Issues](https://bugzilla.cclinux.org/)  |
 | [EuroLinux](https://en.euro-linux.com/)                                        | `eurolinux-*`     | [@nkadel](https://github.com/nkadel)                                  | [Issues](https://github.com/EuroLinux/eurolinux-distro-bugs-and-rfc)  |
 | [Fedora ELN](https://docs.fedoraproject.org/en-US/eln/)                        | `fedora-eln-*`    | [@fedora-eln](https://github.com/fedora-eln)                          | [Issues](https://github.com/fedora-eln/eln/issues)  |
-| [Fedora](https://fedoraproject.org/)                                           | `fedora-*`        | NA                                                                    | [Issues](https://bugzilla.redhat.com/)  |
+| [Fedora](https://fedoraproject.org/)                                           | `fedora-*`        | [@rpm-software-management](https://github.com/rpm-software-management)| [Issues](https://bugzilla.redhat.com/)  |
```

Comment:
This is wrong. There is a specific team that is responsible, not the whole GitHub organization.

Reply:
Any proposal for the change? :)

Reply:
@rpm-software-management/mock-team